GPU/TPCFastTransformation/macro/TPCFastTransformInit.C

```diff
@@ -84,7 +84,7 @@ void TPCFastTransformInit(const char* fileName = "debugVoxRes.root",
 
   o2::tpc::TPCFastSpaceChargeCorrectionHelper* corrHelper = o2::tpc::TPCFastSpaceChargeCorrectionHelper::instance();
 
-  auto corrPtr = corrHelper->createSpaceChargeCorrection(trackResiduals, voxResTree);
+  auto corrPtr = corrHelper->createFromTrackResiduals(trackResiduals, voxResTree);
```

Comment:
Also here I would add to the function call `, bool useSmoothed = false, bool invertSigns = false` and then
```suggest
auto corrPtr = corrHelper->createFromTrackResiduals(trackResiduals, voxResTree, useSmoothed, invertSigns);
```

Reply:
And below again:
```c++
    double correctionX = useSmoothed ? v->DS[o2::tpc::TrackResiduals::ResX] : v->D[o2::tpc::TrackResiduals::ResX];
    double correctionY = useSmoothed ? v->DS[o2::tpc::TrackResiduals::ResY] : v->D[o2::tpc::TrackResiduals::ResY];
    double correctionZ = useSmoothed ? v->DS[o2::tpc::TrackResiduals::ResZ] : v->D[o2::tpc::TrackResiduals::ResZ];

    if (invertSigns) {
      correctionX *= -1.;
      correctionY *= -1.;
      correctionZ *= -1.;
    }
```